tests/Api/Admin/ChannelsTest.php

```diff
@@ -159,109 +151,37 @@ public function it_updates_an_existing_channel(): void
-//                    'postcode' => '12-124',
-//                ],
-//            ], \JSON_THROW_ON_ERROR),
-//        );
-//
-//        $this->assertResponseCode(
-//            $this->client->getResponse(),
-//            Response::HTTP_OK,
-//        );
-//
-//        $this->client->request(
-//            method: 'GET',
-//            uri: sprintf('/api/v2/admin/channels/%s/shop-billing-data', $channel->getCode()),
-//            server: $header,
-//        );
-//
-//        $this->assertResponse(
-//            $this->client->getResponse(),
-//            'admin/shop_billing_data/put_shop_billing_data_response',
-//            Response::HTTP_OK,
-//        );
-//    }
-
     /** @test */
     public function it_deletes_a_channel(): void
     {
+        $this->setUpDefaultDeleteHeaders();
         $this->loadFixturesFromFiles(['authentication/api_administrator.yaml', 'channel.yaml']);
-        $header = array_merge($this->logInAdminUser('api@example.com'), self::CONTENT_TYPE_HEADER);
-
-        $this->client->request(
-            method: 'GET',
-            uri: '/api/v2/admin/channels/MOBILE',
-            server: $header,
-        );
 
+        $this->requestGet('/api/v2/admin/channels/MOBILE');
         $this->assertResponseCode($this->client->getResponse(), Response::HTTP_OK);
 
-        $this->client->request(
-            method: 'DELETE',
-            uri: '/api/v2/admin/channels/MOBILE',
-            server: $header,
-        );
-
+        $this->requestDelete('/api/v2/admin/channels/MOBILE');
         $this->assertResponseCode($this->client->getResponse(), Response::HTTP_NO_CONTENT);
 
-        $this->client->request(
-            method: 'GET',
-            uri: '/api/v2/admin/channels/MOBILE',
-            server: $header,
-        );
-
+        $this->requestGet('/api/v2/admin/channels/MOBILE');
         $this->assertResponseCode($this->client->getResponse(), Response::HTTP_NOT_FOUND);
     }
 
-    // todo: Needs exception_to_status to be investigated
-
-//    /** @test */
-//    public function it_prevents_deleting_the_only_channel(): void
-//    {
-//        $this->loadFixturesFromFiles(['authentication/api_administrator.yaml', 'channel.yaml']);
-//        $header = array_merge($this->logInAdminUser('api@example.com'), self::CONTENT_TYPE_HEADER);
-//
-//        $this->client->request(
-//            method: 'DELETE',
-//            uri: '/api/v2/admin/channels/MOBILE',
-//            server: $header,
-//        );
-//        $this->client->request(
-//            method: 'DELETE',
-//            uri: '/api/v2/admin/channels/WEB',
-//            server: $header,
-//        );
-//
-//        $this->assertResponse(
-//            $this->client->getResponse(),
-//            'admin/channel/delete_channel_that_cannot_be_deleted',
-//            Response::HTTP_UNPROCESSABLE_ENTITY,
-//        );
-//    }
+    /** @test */
+    public function it_prevents_single_channel_deletion(): void
```

Comment:
```suggestion
    public function it_prevents_deleting_the_only_channel(): void
```
better to revert it, `single` sounds like we cannot delete any channel